# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/binaryType-wrong-value.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/binaryType-wrong-value.any.js.ini
--- a/testing/web-platform/meta/websockets/binaryType-wrong-value.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/binaryType-wrong-value.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -6,12 +6,48 @@
 
 [binaryType-wrong-value.any.worker.html?wpt_flags=h2]
   expected:
+    if (os == "linux") and debug and webrender and not swgl and not fission: ["TIMEOUT", "ERROR"]
+    if (os == "linux") and debug and webrender and not swgl and fission: TIMEOUT
+    if (os == "linux") and not debug and not webrender and (processor == "x86_64"): ["OK", "ERROR", "TIMEOUT"]
+    if (os == "linux") and not debug and webrender and fission: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "linux") and not debug and webrender and not fission: ["OK", "TIMEOUT"]
+    if (os == "linux") and debug and webrender and swgl: ["TIMEOUT", "ERROR", "CRASH"]
     if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if os == "linux": TIMEOUT
+    if (os == "linux") and debug and not webrender: ["OK", "TIMEOUT", "ERROR", "CRASH"]
+  [Create WebSocket - set binaryType to something other than blob or arraybuffer - SYNTAX_ERR is returned - Connection should be opened]
+    expected:
+      if (os == "linux") and not fission and not swgl and not debug and not webrender and (processor == "x86_64"): ["PASS", "NOTRUN"]
+      if (os == "linux") and not fission and swgl: NOTRUN
+      if (os == "linux") and fission: ["PASS", "NOTRUN"]
+
+  [Create WebSocket - set binaryType to something other than blob or arraybuffer - SYNTAX_ERR is returned - Connection should be closed]
+    expected:
+      if (os == "linux") and not fission and not swgl and not debug and not webrender and (processor == "x86_64"): ["PASS", "FAIL"]
+      if (os == "linux") and not fission and swgl: FAIL
+      if (os == "linux") and fission: ["PASS", "FAIL"]
+
 
 [binaryType-wrong-value.any.html?wpt_flags=h2]
   expected:
+    if (os == "linux") and debug and not fission and webrender and not swgl: ["TIMEOUT", "OK", "ERROR"]
+    if (os == "linux") and debug and not fission and webrender and swgl: ["OK", "TIMEOUT"]
+    if (os == "linux") and not debug and webrender and fission: ["OK", "TIMEOUT"]
+    if (os == "linux") and not debug and webrender and not fission: ["OK", "ERROR", "TIMEOUT"]
+    if (os == "linux") and not debug and not webrender and (processor == "x86_64"): ["OK", "TIMEOUT"]
+    if (os == "linux") and debug and not fission and not webrender: ["TIMEOUT", "OK"]
+    if (os == "android") and debug and webrender: ["OK", "TIMEOUT"]
+    if (os == "linux") and debug and fission: ["TIMEOUT", "ERROR"]
     if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if os == "linux": TIMEOUT
+  [Create WebSocket - set binaryType to something other than blob or arraybuffer - SYNTAX_ERR is returned - Connection should be opened]
+    expected:
+      if (os == "android") and debug and webrender: ["PASS", "NOTRUN"]
+      if (os == "linux") and (processor == "x86_64"): ["PASS", "NOTRUN"]
+
+  [Create WebSocket - set binaryType to something other than blob or arraybuffer - SYNTAX_ERR is returned - Connection should be closed]
+    expected:
+      if (os == "android") and debug and webrender: ["PASS", "FAIL"]
+      if (os == "linux") and not webrender and (processor == "x86_64"): ["PASS", "FAIL"]
+      if (os == "linux") and webrender: ["PASS", "NOTRUN"]
+
 
 [binaryType-wrong-value.any.html]